label_sync/labels.md

```diff
@@ -173,7 +173,7 @@ larger set of contributors to apply/remove them.
 | <a id="area/mungegithub" href="#area/mungegithub">`area/mungegithub`</a> | Issues or PRs related to code in /mungegithub| label | |
 | <a id="area/planter" href="#area/planter">`area/planter`</a> | Issues or PRs related to code in /planter| label | |
 | <a id="area/prow" href="#area/prow">`area/prow`</a> | Issues or PRs related to prow| label | |
-| <a id="area/prow/artifact-uploader" href="#area/prow/artifact-uploader">`area/prow/artifact-uploader`</a> | Issues or PRs related to prow's artifact-uploader component| label | |
+| <a id="area/prow/artifactuploader" href="#area/prow/artifactuploader">`area/prow/artifactuploader`</a> | Issues or PRs related to prow's artifactuploader component <br><br> This was previously `area/prow/artifact-uploader`, | label | |
```

Comment:
`artifact` and `uploader` are two separate words and it makes sense to have a `-` here in terms of the label.

other examples:
https://github.com/kubernetes/test-infra/blob/1934282eb9f5f0c589922cef5fe82745c1983dd4/label_sync/labels.md
- area/prow/jenkins-operator
- area/prow/pod-utilities
- queue/critical-fix

is there a way to rename the folder but keep the label name?

Reply:
There is, but I think it’s probab good to rename the label to match the package name. That’s the point of these test-infra labels IMO